pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -666,7 +693,11 @@ ${e.message}
     Uri packageConfigUri,
     Uri workingDirectory,
   ) async {
-    final environment = Platform.environment;
+    final environment = _filteredEnvironment({
```

Comment:
I don't think we need to do any filtering when compiling the hook sources to kernel.

Reply:
I would just not pass the environment for hashing - it makes code simpler and I see no downside.